2.3.2/bullseye/Dockerfile

```diff
@@ -162,7 +158,7 @@ COPY include/pip.conf /etc/pip.conf
 COPY include/pip-constraints.txt /usr/local/share/astronomer-pip-constraints.txt
 
 # Pin apache-airflow version to avoid accidental upgrade
-RUN pip freeze | grep "apache-airflow==" >>  /usr/local/share/astronomer-pip-constraints.txt
+RUN echo "apache-airflow==1!${ASTRONOMER_CERTIFIED_VERSION}" >>  /usr/local/share/astronomer-pip-constraints.txt
```

Comment:
Why this change?

Reply:
Because I installed it from the wheel, the command returns `apache-airflow @ file://..`  so I had to do it this way. Now that we would be building the wheel from airflow-private, I will modify this change